draft-ietf-quic-transport.md

```diff
@@ -2510,7 +2510,8 @@ Reset in response, which could lead to an infinite exchange.
 An endpoint MUST ensure that every Stateless Reset that it sends is smaller than
 the packet which triggered it, unless it maintains state sufficient to prevent
 looping.  In the event of a loop, this results in packets eventually being too
-small to trigger a response.
+small to trigger a response.  An endpoint MAY choose not to send a Stateless
+Reset in response to a packet that is smaller than 40 bytes.
```

Comment:
The existing text was confusing, but I don'd see how this text is relevant to looping?

Reply:
This provides the end condition to the loop.